### ISSUE TITLE
Request for consistent UI - Garden Components

### ISSUE THREAD

**Issue by samzx** [zendesk/volunteer_portal#164]
A lot of the interfaces are not consistent. Sometimes using material ui, sometimes bare-bones components.

Might be a big task, but it would be nice to have all components derive from the Zendesk Garden components. https://garden.zendesk.com/

As referenced from: https://github.com/zendesk/volunteer_portal/pull/160#discussion_r286672557

**Comment by samzx** [comment]
depends on #182

**Comment by ThuTrinh** [comment]
### Dialog > Garden Modal
- /components/NamedAvatar.js
- /pages/Dashboard/index.js
- /pages/admin/EventTypes/index.js
- /pages/admin/Offices/index.js
- /pages/admin/Organizations/index.js
- /pages/admin/Tags/index.js
- /pages/admin/Users/index.js
- /pages/MyEvents/index.js
